charts/primary-site/values.yaml

```diff
@@ -11,33 +10,48 @@ globals:
   inbox:
     storageProvider: google_cloud
     bucketName: foxglove-inbox
-
   azure:
     storageAccountName: ""
     ## For example: https://<resourcegroup>.blob.core.windows.net
     serviceUrl: ""
 
 inboxListener:
+  name: inbox-listener
   deployment:
+    scaling:
+      autoscalePollingInterval: 10
+      autoscaleCooldownPeriod: 10
+      minReplicas: 1
+      maxReplicas: 10
+      targetQueueDepth: 2
     resources:
       requests:
         cpu: 1000m
         memory: 1Gi
       limits:
         cpu: 1000m
         memory: 1Gi
+  env:
 
 streamService:
+  name: stream-service
   deployment:
+    scaling:
+      minReplicas: 1
+      maxReplicas: 3
+      targetCPUUtilization: 80
     resources:
       requests:
         cpu: 1000m
         memory: 1Gi
       limits:
         cpu: 1000m
         memory: 1Gi
+  env:
 
 garbageCollector:
+  name: garbage-collector
   schedule: "*/10 * * * *" # every 10 minutes
```

Comment:
should these three items (schedule, failedJobs, successfulJobs) be under a `deployment` section?